distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/config/ConfigureTestCommons.java

```diff
@@ -100,7 +100,7 @@ public static void configureEnforcedMetacardValidators(
   public static Dictionary<String, Object> configureAuthZRealm(
       List<String> attributes, AdminConfig configAdmin) throws IOException {
     Dictionary<String, Object> properties = new DictionaryMap<>();
```

Comment:
❓ Should we be including all of the other properties.  For example, matchAllMappings, etc.?

Reply:
There currently are no itests that use those optional configuration properties

Reply:
This fixes the current issue, but I guess at some point we should update CoinfigureTestCommons.configureAuthZRealm() to configure all of the properties.